wasm_utils/src/api_serialization/get_entry.rs

```diff
@@ -0,0 +1,34 @@
+use holochain_core_types::cas::content::Address;
+
+#[derive(Deserialize, Default, Debug, Serialize)]
+pub struct GetEntryArgs {
+    pub address: Address,
+}
+
+#[derive(Deserialize, Debug, Serialize)]
+pub enum GetResultStatus {
+    Found,
+    NotFound,
+}
+
+#[derive(Deserialize, Debug, Serialize)]
+pub struct GetEntryResult {
+    pub status: GetResultStatus,
+    pub entry: String,
+}
+
+impl GetEntryResult {
+    pub fn found(entry: String) -> GetEntryResult {
+        GetEntryResult {
+            status: GetResultStatus::Found,
+            entry,
+        }
+    }
+
+    pub fn not_found() -> GetEntryResult {
+        GetEntryResult {
+            status: GetResultStatus::NotFound,
+            entry: String::from(""),
+        }
+    }
+}
```

Comment:
This is overall looovely

Reply:
Seems overkill for a boolean

Reply:
Well, the reason why I'm hesitant to put Rust specific Options or Results in here is because this is the go-to place for future HDK implementors to understand what JSON structures they have to parse and create.. and they might not know how and Option gets de-/serialized by serde..

Reply:
I take this all back.  Lets leave the status as is for now (as an enum), because we may use it for added info about CRUD status.  This is fine for MVP.